pom.xml

```diff
@@ -153,6 +153,7 @@
         <module>debezium-server</module>
         <module>debezium-testing</module>
         <module>debezium-connect-rest-extension</module>
+        <module>debezium-api-generator</module>
```

Comment:
Should we maybe call this "Schema Generator". A JSON Schema or OpenAPI are only two of other possible schema formats that could be generated with that generator.

Reply:
Yes, sounds good. And unless you fail physical pain, let's use "schemagenerator" as package name.